lib/spack/docs/basic_usage.rst

```diff
@@ -1461,141 +1334,21 @@ load two or more versions of the same software at the same time.
-that have been included.  The files of the view's installed packages
-are brought into the view by symbolic or hard links back to their
-location in the original Spack installation area.  As the view is
-formed, any clashes due to a file having the exact same path in its
-package installation tree are handled in a first-come-first-served
-basis and a warning is printed.  Packages and their dependencies can
-be both added and removed.  During removal, empty directories will be
-purged.  These operations can be limited to pertain to just the
-packages listed by the user or to exclude specific dependencies and
-they allow for software installed outside of Spack to coexist inside
-the filesystem view tree.
-
-By its nature, a filesystem view represents a particular choice of one
-set of packages among all the versions and variants that are available
-in the Spack installation area.  It is thus equivalent to the
-directory hiearchy that might exist under ``/usr/local``.  While this
-limits a view to including only one version/variant of any package, it
-provides the benefits of having a simpler and traditional layout which
-may be used without any particular knowledge that its packages were
-built by Spack.
-
-Views can be used for a variety of purposes including:
-
-* A central installation in a traditional layout, eg ``/usr/local`` maintained over time by the sysadmin.
-* A self-contained installation area which may for the basis of a top-level atomic versioning scheme, eg ``/opt/pro`` vs ``/opt/dev``.
-* Providing an atomic and monolithic binary distribution, eg for delivery as a single tarball.
-* Producing ephemeral testing or developing environments.
-
-^^^^^^^^^^^^^^^^^^^^^^
-Using Filesystem Views
-^^^^^^^^^^^^^^^^^^^^^^
-
-A filesystem view is created and packages are linked in by the ``spack
-view`` command's ``symlink`` and ``hardlink`` sub-commands.  The
-``spack view remove`` command can be used to unlink some or all of the
-filesystem view.
-
-The following example creates a filesystem view based
-on an installed ``cmake`` package and then removes from the view the
-files in the ``cmake`` package while retaining its dependencies.
-
-.. code-block:: console
-
-   $ spack view --verbose symlink myview cmake@3.5.2
-   ==> Linking package: "ncurses"
-   ==> Linking package: "zlib"
-   ==> Linking package: "openssl"
-   ==> Linking package: "cmake"
-
-   $ ls myview/
-   bin  doc  etc  include  lib  share
-
-   $ ls myview/bin/
-   captoinfo  clear  cpack     ctest    infotocap        openssl  tabs  toe   tset
-   ccmake     cmake  c_rehash  infocmp  ncurses6-config  reset    tic   tput
-
-   $ spack view --verbose --dependencies false rm myview cmake@3.5.2
-   ==> Removing package: "cmake"
-
-   $ ls myview/bin/
-   captoinfo  c_rehash  infotocap        openssl  tabs  toe   tset
-   clear      infocmp   ncurses6-config  reset    tic   tput
-
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Limitations of Filesystem Views
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
The content has been rewritten and integrated into a larger context. See:

^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Discussion: Running Binaries
^^^^^^^^^^^^^^^^^^^^^^^^^^^^


On Thu, Sep 8, 2016 at 11:48 AM, Adam J. Stewart <notifications@github.com>
wrote:

> In lib/spack/docs/basic_usage.rst:
>
> > -   bin  doc  etc  include  lib  share
> > -
> > -   $ ls myview/bin/
> > -   captoinfo  clear  cpack     ctest    infotocap        openssl  tabs  toe   tset
> > -   ccmake     cmake  c_rehash  infocmp  ncurses6-config  reset    tic   tput
> > -
> > -   $ spack view --verbose --dependencies false rm myview cmake@3.5.2
> > -   ==> Removing package: "cmake"
> > -
> > -   $ ls myview/bin/
> > -   captoinfo  c_rehash  infotocap        openssl  tabs  toe   tset
> > -   clear      infocmp   ncurses6-config  reset    tic   tput
> > -
> > -^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> > -Limitations of Filesystem Views
> > -^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> Did you want to delete this section? I don't see it moved anywhere.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LLNL/spack/pull/1676/files/94896e9e3d3aff3e1f727ee895d333f3ff46ec6d#r78033729>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB1cd4L0u3HkhbxEEH467c5sm1N6U7hAks5qoC5ZgaJpZM4JxOaa>
> .
>